Debian 9 - warning: C++11 requires a space

Scanning dependencies of target libpython_pi
[ 53%] Building CXX object plugins/python/CMakeFiles/libpython_pi.dir/cconsole.cpp.o
In file included from /home/dht/src/verlihub-ppa/plugins/python/cconsole.cpp:21:0:
/home/dht/src/verlihub-ppa/plugins/python/cpipython.h:56:31: warning: C++11 requires a space between string literal and macro [-Wc++11-compat]
 #define dprintf(...) { printf("%s:%u: "__FILE__, __LINE__); printf( __VA_ARGS__ ); fflush(stdout); }
                               ^
[ 54%] Building CXX object plugins/python/CMakeFiles/libpython_pi.dir/cpythoninterpreter.cpp.o
In file included from /home/dht/src/verlihub-ppa/plugins/python/cpythoninterpreter.cpp:22:0:
/home/dht/src/verlihub-ppa/plugins/python/cpipython.h:56:31: warning: C++11 requires a space between string literal and macro [-Wc++11-compat]
 #define dprintf(...) { printf("%s:%u: "__FILE__, __LINE__); printf( __VA_ARGS__ ); fflush(stdout); }
                               ^
[ 55%] Building CXX object plugins/python/CMakeFiles/libpython_pi.dir/cpipython.cpp.o
In file included from /home/dht/src/verlihub-ppa/build/plugins/python/cpipython.cpp:21:0:
/home/dht/src/verlihub-ppa/plugins/python/cpipython.h:56:31: warning: C++11 requires a space between string literal and macro [-Wc++11-compat]
 #define dprintf(...) { printf("%s:%u: "__FILE__, __LINE__); printf( __VA_ARGS__ ); fflush(stdout); }
                               ^

--- a/plugins/python/cpipython.h
+++ b/plugins/python/cpipython.h
@@ -53,7 +53,7 @@
 #define log4(...) { if (cpiPython::log_level > 3) { printf( __VA_ARGS__ ); fflush(stdout); }; }
 #define log5(...) { if (cpiPython::log_level > 4) { printf( __VA_ARGS__ ); fflush(stdout); }; }
 
-#define dprintf(...) { printf("%s:%u: "__FILE__, __LINE__); printf( __VA_ARGS__ ); fflush(stdout); }
+#define dprintf(...) { printf("%s:%u: " __FILE__, __LINE__); printf( __VA_ARGS__ ); fflush(stdout); }
 
 using std::vector;
 namespace nVerliHub {

